src/main/java/pl/senla/hotel/controller/ControllerOrderSpring.java

```diff
@@ -0,0 +1,75 @@
+package pl.senla.hotel.controller;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import pl.senla.hotel.dto.OrderCreateDto;
+import pl.senla.hotel.dto.OrderReadDto;
+import pl.senla.hotel.entity.Order;
+import pl.senla.hotel.entity.services.HotelService;
+import pl.senla.hotel.service.ServiceOrder;
+import pl.senla.hotel.service.ServiceRoomReservation;
+import pl.senla.hotel.utils.OrderReadDtoMapperUtil;
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/orders")
+@Slf4j
+public class ControllerOrderSpring implements ControllerOrder {
+
+    @Autowired
+    private ServiceOrder orderService;
+    @Autowired
+    private ServiceRoomReservation serviceRoomReservation;
+
+    @Override
+    @GetMapping
+    public List<OrderReadDto> readAll() {
+        log.debug("ControllerOrder call ServiceOrder's method 'readAll'.");
+        return OrderReadDtoMapperUtil
+                .convertListOrderToOrderDtoRead(orderService.readAll(), serviceRoomReservation.readAll());
```

Comment:
you don't need to do this because each order will have all the attached services inside

Reply:
ok. ready